docs/sources/next/results-output/real-time/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'OpenTelemetry'
+description: 'Use the OpenTelemetry output to send test results in OTEL metrics format.'
+weight: 00
+---
+
+# OpenTelemetry
+
+k6 can push test run metrics in a [OpenTelemetry metrics format](https://opentelemetry.io/docs/specs/otel/metrics/) to a OTEL metrics collector or a metrics backend that support OTEL metrics fromat by using the [xk6-output-opentelemetry extension](https://github.com/grafana/xk6-output-opentelemetry/).
+
+This is an extension, and we are looking for feedback and contributions and aiming to graduate it as an experimental output within k6 core in one of the future releases. Feel free to open an issue directly in the [grafana/xk6-output-opentelemetry repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
+
+## Metrics mapping
+
+All [k6 metric types](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/metrics) are converted into an equivalent [OpenTelemetry metric type](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#timeseries-model). The k6's tags converted to the OpenTelemetry attributes. Whenever it's possible, the units also passed with the metrics.
```

Comment:
```suggestion
k6 converts all [k6 metric types](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/metrics) to an equivalent [OTEL metric type](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#timeseries-model), and all k6 tags to OTEL attributes. When possible, the units are also passed with the metrics.
```

I'm not 100% sure on the meaning of the last sentence, would you mind explaining it to me? 🤓